namui-user-config/src/set_user_config.rs

```diff
@@ -33,11 +33,15 @@ pub fn set_namui_user_config(target: &Target) -> Result<(), Box<dyn std::error::
     };
 
     fs::write(
-        &namui_user_config_path,
+        &user_config_path,
         &serde_json::to_string_pretty(&namui_user_config)
             .map_err(|error| format!("namui user config stringify error: {}", error))?,
     )?;
-
-    println!("Settings have been saved.");
     Ok(())
 }
+
+fn ensure_user_config_dir(user_config_path: &PathBuf) -> Result<(), std::io::Error> {
+    let mut user_config_dir = user_config_path.clone();
+    user_config_dir.pop();
+    create_dir_all(user_config_dir)
```

Comment:
What's going in this code? why pop? to get directory path, not file path?

Reply:
Yes, ensuring user_config.json located directory